slider/replay.py

```diff
@@ -152,6 +154,64 @@ def _consume_actions(buffer):
     return out
 
 
+def _ms(t):
+    return t.total_seconds() * 1e3
+
+
+def _within(p1, p2, d):
```

Comment:
can you add a numpy-style docstring (the format used throughout the project) here explaining what `p1`, `p2`, and `d` are?